core/client.go

```diff
@@ -13,14 +13,14 @@ import (
 type Client = client.Client
 
 // NewRemote creates a new Client that communicates with a remote Core endpoint over HTTP.
-func NewRemote(protocol, remoteAddr string) (Client, error) {
+func NewRemote(endpoint string) (Client, error) {
```

Comment:
I would make this constructor receive a port with an IP as a separate params, as you do now in CoreAccessor.

Reply:
https://github.com/celestiaorg/celestia-node/pull/910/commits/5889004444a76ba75580b645cc785cadb69cac71